testing/cockroach/cockroach.confd

```diff
@@ -0,0 +1 @@
+cockroach_opts="start --insecure --store=/var/lib/cockroach"
```

Comment:
`start` is not really optional, is it? I think that can better be moved to the initd `command_args`.

Reply:
Well, I put it there rather than command_args so that the 'init' command for initialising a cluster rather than starting a standalone node can easily be substituted without touching init scripts.

Reply:
Alright, makes sense.